### PR TITLE
update template

### PR DESCRIPTION
**Pull request by thepomeranian**

### CODE REVIEW

File: TEMPLATE.md [hunk 1]
```diff
@@ -17,62 +17,82 @@ requires (*optional): <AIP number(s)>
...
 
-Include a brief description summarizing the intended change. This should be no more than a couple of sentences. Discuss the business impact and business value this change would impact.
+- Summarize in 3-5 sentences what is the problem we’re solving for and how are we solving for it
 
 ## Motivation
 
-Describe the impetus for this change. What does it accomplish? What might occur if we do not accept this proposal?
+### Goals
+
+- What are the goals and what is in scope? Any KPI’s/metrics?
```
Comment (movekevin) [inline TEMPLATE.md:26]: nit: KPI's -> KPIs?

File: TEMPLATE.md [hunk 2]
```diff
@@ -17,62 +17,82 @@ requires (*optional): <AIP number(s)>
...
 
 ## Security Considerations
 
-Has this change being audited by any auditing firm? 
-Any potential scams? What are the mitigation strategies?
-Any security implications/considerations?
-Any security design docs or auditing materials that can be shared?
-
-## Testing (optional)
+- Has this change being audited by an auditing firm?
```
Comment (movekevin) [inline TEMPLATE.md:90]: Has this change been audited

File: TEMPLATE.md (hunk 1 above)
Comment (thepomeranian) [reply TEMPLATE.md]: fixed!

File: TEMPLATE.md [hunk 3]
```diff
@@ -17,62 +17,82 @@ requires (*optional): <AIP number(s)>
...
-Include a brief description summarizing the intended change. This should be no more than a couple of sentences. Discuss the business impact and business value this change would impact.
+- Summarize in 3-5 sentences what is the problem we’re solving for and how are we solving for it
 
 ## Motivation
 
-Describe the impetus for this change. What does it accomplish? What might occur if we do not accept this proposal?
+### Goals
+
+- What are the goals and what is in scope? Any KPI’s/metrics?
+- What is the business impact and value of this solution (immediate and long term future)?
```
[the remark this answers is not in the document]
Comment (thepomeranian) [reply TEMPLATE.md]: fixed

File: TEMPLATE.md (hunk 2 above)
Comment (thepomeranian) [reply TEMPLATE.md]: fixed

**Review by davidiw — COMMENTED** [review]
merged as https://github.com/aptos-foundation/AIPs/pull/231

**Pull request closed without merging**